packages/workflow-diagram/src/types.ts

```diff
@@ -2,12 +2,14 @@ import { Edge, Node } from "react-flow-renderer";
 
 export type CronTrigger = {
   type: "cron";
+  expression: string;
```

Comment:
Could you change this to `cron_expression` and `url` to `webhook_url`.

While these fields look like they are their own interfaces, they are types that are unioned: `export type Trigger = CronTrigger | WebhookTrigger | FlowTrigger;`

So it's important we use more specific names, I suggest sticking to the same name we use on Lightning, i.e. `cron_expression` etc. 

This change will need to be done in https://github.com/OpenFn/Lightning/pull/412 as well.

Reply:
Done. Instead of snake case naming (`cron_expression` and `webhook_url`) I did chose camel case naming (`cronExpression` and `webhookUrl`) to respect our naming in Kit. I have also updated the Lightning side (see [here](https://github.com/OpenFn/Lightning/pull/412/commits/96fb03510c4f7d19f32bd4710dd2a4d7bf5de7fb))